test/compiler.test.js

```diff
@@ -1016,41 +1016,106 @@ describe('compiler', function() {
     });
 
     describe('for mixins', function() {
-      function verifyMixinIsFoundViaMixinDirs(sourceFile, mixinDirs) {
-        var appJS = appdir.writeFileSync(sourceFile, '');
+      describe(' - mixinDirs', function() {
+        function verifyMixinIsFoundViaMixinDirs(sourceFile, mixinDirs) {
+          var appJS = appdir.writeFileSync(sourceFile, '');
 
-        var instructions = boot.compile({
-          appRootDir: appdir.PATH,
-          mixinDirs: mixinDirs
+          var instructions = boot.compile({
+            appRootDir: appdir.PATH,
+            mixinDirs: mixinDirs
+          });
+
+          expect(instructions.mixins[0].sourceFile).to.eql(appJS);
+        }
+
+        it('supports `mixinDirs` option', function() {
+          verifyMixinIsFoundViaMixinDirs('mixins/other.js', ['./mixins']);
         });
 
-        expect(instructions.mixins[0].sourceFile).to.eql(appJS);
-      }
+        it('resolves relative path in `mixinDirs` option', function() {
+          verifyMixinIsFoundViaMixinDirs('custom-mixins/other.js',
+            ['./custom-mixins']);
+        });
 
-      it('supports `mixinDirs` option', function() {
-        verifyMixinIsFoundViaMixinDirs('mixins/other.js', ['./mixins']);
+        it('resolves module relative path in `mixinDirs` option', function() {
+          verifyMixinIsFoundViaMixinDirs('node_modules/custom-mixins/other.js',
+            ['custom-mixins']);
+        });
       });
 
-      it('resolves relative path in `mixinDirs` option', function() {
-        verifyMixinIsFoundViaMixinDirs('custom-mixins/vehicle.js',
```

Comment:
It was not fitting the 80 character limit. So, instead of moving ```custom-mixins/vehicle.js``` to new line, I renamed ```vehicle.js``` to ```other.js```